build.sbt

```diff
@@ -61,7 +61,8 @@ lazy val commonSettings = List(
     "-doc-title",
     "Bitcoin-S",
     "-doc-version",
-    version.value
+    version.value,
+    "-no-link-warnings"
```

Comment:
Shouldn't add this IMO

Reply:
I have removed it for this PR just so we have some of the added scaladocs where they didn't already exist. I'll leave a note about this in my comment at the top but I agree for now we leave this out of this PR and settle it later.